core/src/main/java/io/grpc/internal/MessageFramer.java

```diff
@@ -296,6 +352,17 @@ private void writeRaw(byte[] b, int off, int len) {
    */
   @Override
   public void flush() {
+    if (gzipOutputStream != null) {
+      if (gzipCloseOnFlush) {
+        closeGzipOutputStream();
+      } else {
+        try {
+          gzipOutputStream.flush();
+        } catch (IOException e) {
```

Comment:
It's weird this exception is "handled" but thrown away. This seems like it should be logged at warning, right?